apps/web/src/pages/ApplicantDashboardPage/components/CareerDevelopmentTaskCard.tsx

```diff
@@ -0,0 +1,276 @@
+import { useIntl } from "react-intl";
+import Cog8ToothIcon from "@heroicons/react/24/outline/Cog8ToothIcon";
+
+import {
+  FragmentType,
+  getFragment,
+  MoveInterest,
+  OrganizationTypeInterest,
+} from "@gc-digital-talent/graphql";
+import {
+  Accordion,
+  AccordionMetaData,
+  PreviewList,
+  TaskCard,
+  Well,
+} from "@gc-digital-talent/ui";
+
+import useRoutes from "~/hooks/useRoutes";
+
+import MoveInterestsList from "./MoveInterestsList";
+import OrganizationTypeInterestsList from "./OrganizationInterestsList";
+import FunctionalCommunityListItem, {
+  PreviewListItemFunctionalCommunity_Fragment,
+} from "./FunctionalCommunityListItem";
+
+interface CareerDevelopmentTaskCardProps {
+  moveInterests: MoveInterest[] | null;
+  organizationTypeInterests: OrganizationTypeInterest[] | null;
+  communityInterestsQuery: NonNullable<
+    FragmentType<typeof PreviewListItemFunctionalCommunity_Fragment>
+  >[];
+}
```

Comment:
I think typing this will be easier if we build fragments up from the nested components rather than drilling down. Not sure if that makes sense but essentially:

```tsx
const CareerDevelopementTaskCard_Fragment = graphql(
   fragment CareedDevelopmentCard on EmployeeProfile {
       moveInterests
       organziationTypeInterests
      ...PreviewListItemFunctionCommunity
   }
);

interface CareerDevelopmentTaskCardProps {
  query: FragmentType<CareerDevelopmentTaskCard_Fragment>;
}
```

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/12607/commits/dff254d46efbfb5f6a266186d3eff757e12f4239